app/components/Builder.tsx

```diff
@@ -557,8 +557,7 @@ export const Builder = (props: BuilderProps & AuthProps): JSX.Element => {
       isPrivate: state.isPrivate,
       isPrivateUntil: state.isPrivateUntil?.toMillis(),
       alternates: state.alternates,
-      userTags: state.userTags,
-      symmetry: state.symmetry,
```

Comment:
Was this just an accidental change? I think the effect is to not store the current symmetry setting between sessions.

Reply:
forgot to revert that as well from when I was using <kbd>F5</kbd> as a symmetry reset button